doc/nvim-treesitter.txt

```diff
@@ -97,5 +97,16 @@ A list of languages can be found at |:TSInstallInfo|
 
 List modules state for the current session.
 
- vim:tw=78:ts=8:noet:ft=help:norl:
+==============================================================================
+FUNCTIONS~
+						   *nvim-treesitter-functions*
+
+|require'nvim-treesitter'.statusline(size)|
```

Comment:
didn't you provide a vim function ?

Reply:
Oh yeah you are right, completely forgot about it...